ui/app/components/app-header/app-header.component.js

```diff
@@ -158,36 +152,9 @@ export default class AppHeader extends PureComponent {
           </div>
           <div>
             {cashaccount && cashaccount.information !== undefined ? (
-              <Tooltip
```

Comment:
does this need to be removed for dividend txs?

Reply:
No that's from a merge, thanks!